packages/gatsby/src/utils/webpack.config.js

```diff
@@ -430,7 +415,7 @@ module.exports = async (
     // https://github.com/defunctzombie/package-browser-field-spec); setting
     // the target tells webpack which file to include, ie. browser vs main.
     target: stage === `build-html` || stage === `develop-html` ? `node` : `web`,
-    profile: stage === `production`,
```

Comment:
i don't think this was ever `true`? stage can't be `production`

Reply:
haha yeah 🤔